src/store/fileUploadStore.js

```diff
@@ -428,7 +428,11 @@ const actions = {
 	 * @param {object|null} data.options The share options
 	 */
 	async shareFiles(context, { token, uploadId, lastIndex, caption, options }) {
-		const performShare = async ([index, shareableFile]) => {
+		const performShare = async (share) => {
+			if (!Array.isArray(share)) {
+				return
+			}
```

Comment:
Is it known in which case it is not an array?

Reply:
problem was when we're trying to call it with arg `undefined`. only `Object.entries` entries should be handled here

Reply:
It's when upload fails